assesspy/__init__.py

```diff
@@ -14,11 +16,5 @@
     prd,
     prd_met,
 )
-from .load_data import ratios_sample
-from .outliers import (
-    iqr_outlier,
-    is_outlier,
-    quantile_outlier,
-)
-from .sales_chasing import detect_chasing
-from .utils import check_inputs
+from .outliers import is_outlier
+from .sales_chasing import is_sales_chased
```

Comment:
I decided to hide of some of the exported functions here (iqr_outlier, quantile_outlier) to keep things in line with how `is_sales_chased` works. Both function are accessible via `is_outlier` anyways.